articles/resources/relnotes.md

```diff
@@ -17,6 +17,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: November 23rd, 2020*
+
+- Improved compiler performance with faster reference loading.
+- Updated Q# grammar for language repository.Fixed lexer bug with double literals and ranges. Updated grammar for namespaces, namespace elements, statements, and types and labels are used consistently. 
+- Updated Microsoft.Quantum.Preparation API.
+- Q# ergonomic improvements: eliminate the need for parentheses around subsequent calls. 
+- Updated IQ# client telemetry to use the latest Microsoft Cookie Compliance API and GDPR cookie consent banner. This requires explicit user consent for non-critical cookies, and the user has a finer control on how cookies can be used. More here: https://www.1eswiki.com/wiki/WCP_Cookie_Consent_API.
+- Allocated qubits not reused when optimizing for depth. When the QCTraceSimulator is set for OptimizeDepth = true, then the width of SimultaneousUse should be 20, and depth (of all gates) should be 4. SequentialUse should have depth 24 and also have width 20.
```

Comment:
Since it sounds from this description like this change is only applicable to Q# + .NET users, can we note that explicitly and link to the relevant .NET API Browser entries for each class and parameter? (On a side note, we may want to find a way to expose this functionality to Python users as well, since resource estimation is a pretty major usecase for Q# + Python interoperability.)

Reply:
This was not completed for this month's release  and it needs more maturing. I'll remove it.